extensionsGallery-insider.json

```diff
@@ -5511,6 +5511,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "109",
+					"extensionName": "couchbase",
+					"displayName": "Couchbase",
+					"shortDescription": "Welcome to the official Azure Data Studio extension for Couchbase! This extension is designed to provide a seamless experience for Couchbase Server users who want to work within the popular Visual Studio Code editor.",
+					"publisher": {
+						"displayName": "Aayush Tyagi",
+						"publisherId": "AayushTyagi1",
+						"publisherName": "Aayush Tyagi"
+					},
+					"versions": [
+						{
+							"version": "2.1.4",
+							"lastUpdated": "09/03/2024",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/Couchbase-Ecosystem/VSCode-Couchbase/releases/latest"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://github.com/Couchbase-Ecosystem/VSCode-Couchbase/blob/main/images/couchbase.png"
```

Comment:
The icon needs to be hosted on `https://raw.githubusercontent.com/` instead of `https://github.com/`